fib-prime.js

```diff
@@ -1,34 +1,35 @@
-var ispnum = function(num) {
-  for(var i = 2; i < num; i++)
-    if  (num % i === 0) return false;
-  return num > 1;
-};
+const isPrime = num => {
+  for (let i = 2, s = Math.sqrt(num); i <= s; i++)
+    if (num % i === 0) return false
 
-const fibonacci = (num) => {
-  if (num <= 1)return 1;
-  return fibonacci(num - 1) + fibonacci(num - 2);
-};
+  return num > 1
+}
+
+const fibonacci = num => {
+  if (num <= 1) return 1
+  return fibonacci(num - 1) + fibonacci(num - 2)
+}
+
+const nextPrimeFibonacci = input => {
+  let iterable = 1
 
-function nxtPrmFib(number) {
-  let r = 0;
-  let l = 1;
   while (true) {
-    var fib = fibonacci(l);
-    console.log('fib', fib, number);
-    if (fib > number) {
-      if (ispnum(fib)) {
-        r = fib;
-        break;
+    let result = fibonacci(iterable)
+    console.log('input value:', input, 'current result:', result)
+
+    if (result > input) {
+      if (isPrime(result)) {
+        return result
```

Comment:
FIXED: Changed to return result from function. The instruction says about the function `returns a number`.